### PR TITLE
Clean up Bikeshed warnings

### LINKED ISSUE #1224: Clean up Bikeshed warnings
As of commit db94f4a72567ede161a26482c8ba3292e737b70d with latest Bikeshed version on 2019-05-31:

```
WARNING: You should manually provide IDs for your headings:
  <h3 line-number="2311" class="heading" data-level="5.7" id="webauthn-extensions-inputs-and-outputs"><span class="content">WebAuthn Extensions Inputs and Outp
uts</span></h3>
LINK ERROR: Multiple possible 'browsing context' dfn refs.
Arbitrarily chose https://html.spec.whatwg.org/multipage/browsers.html#browsing-context
To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
spec:html; type:dfn; for:/; text:browsing context
spec:html; type:dfn; for:Document; text:browsing context
spec:html; type:dfn; for:Window; text:browsing context
[=browsing context=]
LINK ERROR: Multiple possible 'feature policy' dfn refs.
Arbitrarily chose https://html.spec.whatwg.org/multipage/infrastructure.html#concept-feature-policy
To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
spec:html; type:dfn; for:Document; text:feature policy
The following refs show up multiple times in their spec, in a way that Bikeshed can't distinguish between. Either create a manual link, or ask the spec maintai
ner to add disambiguating attributes (usually a for='' attribute to all of them).
spec:html; type:dfn; for:/; text:feature policy
  https://html.spec.whatwg.org/multipage/infrastructure.html#concept-feature-policy
  https://html.spec.whatwg.org/multipage/infrastructure.html#concept-policy-controlled-feature
[=feature policy=]
LINK ERROR: No 'dfn' refs found for 'domexception' that are marked for export.
[=DOMException=]
WARNING: The following locally-defined biblio entries are unused and can be removed:
  * geojson
  * oscca-sm3
  * sp800-107r1
WARNING: Unexported dfn that's not referenced locally - did you mean to export it?
<dfn data-dfn-for="CreateCred-AuthnrSuccess" data-dfn-type="dfn" id="createcred-authnrsuccess-indicates-success" data-lt="indicates success" data-noexport="by-
default">indicates success<a class="self-link" href="#createcred-authnrsuccess-indicates-success"></a></dfn>
WARNING: Unexported dfn that's not referenced locally - did you mean to export it?
<dfn data-lt="webauthn-feature" data-dfn-type="dfn" id="webauthn-feature" data-noexport="by-default">webauthn<a class="self-link" href="#webauthn-feature"></a>
</dfn>
WARNING: Unexported dfn that's not referenced locally - did you mean to export it?
<dfn data-dfn-type="dfn" id="second-factor-platform-authenticator" data-lt="Second-factor platform authenticator" data-noexport="by-default">Second-factor plat
form authenticator<a class="self-link" href="#second-factor-platform-authenticator"></a></dfn>
WARNING: Unexported dfn that's not referenced locally - did you mean to export it?
<dfn data-dfn-type="dfn" id="first-factor-platform-authenticator" data-lt="First-factor platform authenticator" data-noexport="by-default">First-factor platfor
m authenticator<a class="self-link" href="#first-factor-platform-authenticator"></a></dfn>
WARNING: Unexported dfn that's not referenced locally - did you mean to export it?
<dfn data-dfn-type="dfn" id="second-factor-roaming-authenticator" data-lt="Second-factor roaming authenticator" data-noexport="by-default">Second-factor roamin
g authenticator<a class="self-link" href="#second-factor-roaming-authenticator"></a></dfn>
WARNING: Unexported dfn that's not referenced locally - did you mean to export it?
<dfn data-dfn-type="dfn" id="user-verifying-roaming-authenticator" data-lt="User-verifying roaming authenticator" data-noexport="by-default">User-verifying roa
ming authenticator<a class="self-link" href="#user-verifying-roaming-authenticator"></a></dfn>
WARNING: Unexported dfn that's not referenced locally - did you mean to export it?
<dfn data-dfn-type="dfn" id="resident-key-credential-property" data-lt="resident key credential property" data-noexport="by-default">resident key credential pr
operty<a class="self-link" href="#resident-key-credential-property"></a></dfn>
 ✔  Successfully generated, with 3 linking errors
==============DONE==============
```

### PR DESCRIPTION
**Pull request by emlun**
Will fix #1224 once completed.

Notes:

- The last reference to `GeoJSON` was removed in commit 1fc8906a20bb0698d68de3fbe55ebd109617b3bd
- The last reference to `SP800-107r1` was removed in commit c42a822a357c1697c999d930106ed666f78261c5
- The last reference to `OSCCA-SM3` was removed in commit c42a822a357c1697c999d930106ed666f78261c5

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1225.html" title="Last updated on Jun 10, 2019, 10:54 AM UTC (fb9337f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/1225/db94f4a...fb9337f.html" title="Last updated on Jun 10, 2019, 10:54 AM UTC (fb9337f)">Diff</a>

### CODE REVIEW

**Comment by emlun** [comment]
@equalsJeffH Please advise on the remaining 4 issues:

- [ ] Disambiguate imported definition `[=browsing context=]`

  ```
  LINK ERROR: Multiple possible 'browsing context' dfn refs.
  Arbitrarily chose https://html.spec.whatwg.org/multipage/browsers.html#browsing-context
  To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
  spec:html; type:dfn; for:/; text:browsing context
  spec:html; type:dfn; for:Document; text:browsing context
  spec:html; type:dfn; for:Window; text:browsing context
  [=browsing context=]
  ```

- [ ] Disambiguate imported definition `[=feature policy=]`

  ```
  LINK ERROR: Multiple possible 'feature policy' dfn refs.
  Arbitrarily chose https://html.spec.whatwg.org/multipage/infrastructure.html#concept-feature-policy
  To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
  spec:html; type:dfn; for:Document; text:feature policy
  The following refs show up multiple times in their spec, in a way that Bikeshed can't distinguish between. Either create a manual link, or ask the spec maintainer to add disambiguating attributes (usually a for='' attribute to all of them).
  spec:html; type:dfn; for:/; text:feature policy
    https://html.spec.whatwg.org/multipage/infrastructure.html#concept-feature-policy
    https://html.spec.whatwg.org/multipage/infrastructure.html#concept-policy-controlled-feature
  [=feature policy=]
  ```

- [ ] Export or un-`<dfn>` the term `indicates success`

  It looks to me like the purpose of this is create a hyperlink anchor, is that correct?

  ```
  WARNING: Unexported dfn that's not referenced locally - did you mean to export it?
  <dfn data-dfn-for="CreateCred-AuthnrSuccess" data-dfn-type="dfn" id="createcred-authnrsuccess-indicates-success" data-lt="indicates success" data-noexport="by-default">indicates success<a class="self-link" href="#createcred-authnrsuccess-indicates-success"></a></dfn>
  ```

- [ ] Export the term `webauthn-feature`?

  ```
  WARNING: Unexported dfn that's not referenced locally - did you mean to export it?
  <dfn data-lt="webauthn-feature" data-dfn-type="dfn" id="webauthn-feature" data-noexport="by-default">webauthn<a class="self-link" href="#webauthn-feature"></a></dfn>
  ```

**Comment by emlun** [comment]
On further inspection it looks like the `indicates success` definition was added in a draft of the `credProps` extension, and the reference to it was later removed in commit 2fe490c1effeb6accf38bee541945cd43e8fac5e. So that one should be safe to delete.

**Comment by equalsJeffH** [comment]
> On further inspection it looks like the `indicates success` definition was added in a draft of the `credProps` extension, and the reference to it was later removed...

OK

File: index.bs [hunk 1]
```diff
@@ -1794,7 +1799,7 @@ This method has no arguments and returns a Boolean value.
     };
 </xmp>
 
-Note: Invoking this method from a [=browsing context=] where the [=Web Authentication API=] is "disabled" according to the [=allowed to use=] algorithm&mdash;i.e., by a [=feature policy=]&mdash;will result in the promise being rejected with a [=DOMException=] whose name is "{{NotAllowedError}}".
+Note: Invoking this method from a [=browsing context=] where the [=Web Authentication API=] is "disabled" according to the [=allowed to use=] algorithm&mdash;i.e., by a [=feature policy=]&mdash;will result in the promise being rejected with a {{DOMException}} whose name is "{{NotAllowedError}}".
```
Comment (equalsJeffH) [inline index.bs]: Good catch, thx :)

File: index.bs [hunk 2]
```diff
@@ -2308,7 +2313,7 @@ SHOULD be aborted.
     See [WHATWG HTML WG Issue #2711](https://github.com/whatwg/html/issues/2711) for more details. 
 
 
-## WebAuthn Extensions Inputs and Outputs
+## WebAuthn Extensions Inputs and Outputs ## {#sctn-extensions-inputs-outputs}
```
Comment (equalsJeffH) [inline index.bs]: oops, thx :)

File: index.bs [hunk 3]
```diff
@@ -2830,19 +2835,19 @@ the same procedure as other [=assertion signatures=] generated by the [=authenti
...
             <tr>
-                <th> <dfn>Second-factor roaming authenticator</dfn> </th>
+                <th> Second-factor roaming authenticator </th>
                 <td> [=cross-platform attachment|cross-platform=] </td>
                 <td> [=server-side credential storage modality|Server-side storage=] </td>
                 <td> [=single-factor capable|Single-factor=] </td>
             </tr>
             <tr>
-                <th> <dfn>User-verifying roaming authenticator</dfn> </th>
+                <th> User-verifying roaming authenticator </th>
```
Comment (equalsJeffH) [inline index.bs]: I'm tempted to leave these dfns here and export them. In any case, we ought to be exporting a fair number of our dfns I'm thinking. see also issue #1049.

File: index.bs [hunk 4]
```diff
@@ -5097,6 +5102,17 @@ At this time, one [=credential property=] is defined. It is known abstractly as
...
 
+    <div dfn-type="dict-member" dfn-for="CredentialPropertiesOutput">
+        :   <dfn>rk</dfn>
+        ::  This OPTIONAL property, known abstractly as the <dfn dfn-type="dfn">resident key credential property</dfn>,
+            is a Boolean value indicating whether the {{PublicKeyCredential}} returned as a result of a [=registration ceremony=]
+            is a [=resident credential=].
+            If {{rk}} is [TRUE], the credential is a [=resident credential=];
+            if {{rk}} is [FALSE], the credential is a [=non-resident credential=].
+            if {{rk}} is [=not present=], it is not known whether the credential is a [=resident credential=] or a [=non-resident credential=].
+    </div>
```
Comment (equalsJeffH) [inline index.bs]: nice!

File: index.bs (hunk 1 above)
Comment (emlun) [reply index.bs]: Thank Bikeshed, not me :smile:

File: index.bs (hunk 3 above)
Comment (emlun) [reply index.bs]: I'm not sure where these definitions would ever be used outside this spec. How about removing them for now and instead documenting in #1049 that we might want to consider re-introducing them later?

Comment (equalsJeffH) [reply index.bs]: sure :)

**Pull request merged**